jobs/JGLOBAL_WAVE_PREP

```diff
@@ -77,6 +77,10 @@ else
   if [ ! -L $ROTDIR/${WAVECUR_DID}.${RPDY} ]; then # Check if symlink already exists in ROTDIR
     $NLN $DMPDIR/${WAVECUR_DID}.${RPDY} $ROTDIR/${WAVECUR_DID}.${RPDY}
   fi
+  BRPDY=`$NDATE -24 ${RPDY}00 | cut -c1-8`
```

Comment:
Can we start the transition away from backticks as we make any modifications?

Reply:
Actually, don't worry about all the backticks. I'm going to just go through and replace them in all scripts after this PR is merged (though I will need someone to test afterwards).